docs/machine-learning/automate-training-with-model-builder.md

```diff
@@ -13,11 +13,17 @@ Model Builder uses automated machine learning (AutoML) to explore different mach
 
 You don't need machine learning expertise to use Model Builder. All you need is some data, and a problem to solve. Model Builder generates the code to add the model to your .NET application.
 
-![Model Builder Visual Studio extension user interface animation](media/ml-dotnet-model-builder.gif)
+![Model Builder Scenarios](./media/model-builder-scenarios.png)
 
 > [!NOTE]
 > Model Builder is currently in Preview.
 
+## Creating a Model Builder Project
```

Comment:
I would expand on this a bit with this information

https://devblogs.microsoft.com/dotnet/ml-net-and-model-builder-march-updates/

```markdown
After training, two files are generated under the *.mbconfig file:

- Model.consumption.cs: This file contains the Model Input and Model Output schemas as well as the Predict function generated for consuming the model.
- Model.training.cs: This file contains the training pipeline (data transforms, algorithm, algorithm hyperparameters) chosen by Model Builder to train the model. You can use this pipeline for re-training your model.
- Model.zip: This is a serialized zip file which represents your trained ML.NET model.
```